test/typescript-karma/karma-conf.js

```diff
@@ -3,17 +3,17 @@
 const fs = require('fs');
 const path = require('path');
 
-function sineData (_request, response, _next) {
-    const csv = [[ 'X', 'sin(n)', 'sin(-n)' ]];
+// function sineData (_request, response, _next) {
```

Comment:
Can we remove that or will it be used later?

Reply:
I am not sure. I copied the code to karma-fetch as it was not working with fetch. Though I guess I can delete it.